<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>761</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addVertexBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Vertex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addEdgeBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Edge</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startCalcalationBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>560</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showPathBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>560</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stepBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randGenBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>560</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Generate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
